Raygun.Druid4Net/Fluent/Context/IContextSpec.cs

```diff
@@ -79,5 +79,10 @@ public interface IContextSpec
     /// </summary>
     /// <remarks>default is false</remarks>
     bool? SerializeDateTimeAsLongInner { get; set; }
+    
+    /// <summary>
+    /// Overrides the brokers search strategy when a search query is performed.
+    /// </summary>
+    SearchStrategy? Strategy { get; set; }
```

Comment:
Since this is specific to a Search query this should be on a specific context that applies the the Search query. Have a look at how the Timeseries query has a specific `TimeseriesContextSpec`. I think we'll need to create a `SearchContextSpec` for this property.

Also, I think the property name for this should be `SearchStrategy` according to these docs https://druid.apache.org/docs/latest/querying/searchquery.html#query-context